swagger.yaml

```diff
@@ -70,7 +70,7 @@ paths:
           type: string
         - in: query
```

Comment:
What is your collectionFormat for this query? We originally talked about doing "multi" (ex. ?type=dining&type=building) but I wasn't sure if that's what you're implementing.